multi-agent/tests/e2e/test_session_stress_submit.py

```
@@ -0,0 +1,1442 @@
+    if stress_config.blueprint_id:
+        return None
+
+    if stress_config.blueprint_path:
+        from pathlib import Path
+        
+        blueprint_file = Path(stress_config.blueprint_path)
+        if not blueprint_file.exists():
+            pytest.skip(f"Blueprint file not found: {stress_config.blueprint_path}")
+        
+        print(f"📄 Loading blueprint from: {stress_config.blueprint_path}")
+        with open(blueprint_file, 'r') as f:
```

Ruff (0.16.1)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)